plugin/src/main/cpp/classes/openxr_hybrid_app.cpp

```diff
@@ -46,26 +52,34 @@ void OpenXRHybridApp::_bind_methods() {
 	BIND_ENUM_CONSTANT(HYBRID_MODE_PANEL);
 }
 
-bool OpenXRHybridApp::is_hybrid_app() const {
-	OS *os = OS::get_singleton();
-	ERR_FAIL_NULL_V(os, false);
+bool OpenXRHybridApp::is_hybrid_app_enabled() const {
+	return ProjectSettings::get_singleton()->get_setting_with_override("xr/hybrid_app/enabled");
+}
 
-	return os->has_feature("godot_openxr_hybrid_app");
+bool OpenXRHybridApp::is_hybrid_app() const {
+#ifdef ANDROID_ENABLED
+	return OS::get_singleton()->has_feature("godot_openxr_hybrid_app");
+#else
+	Ref<OpenXRInterface> xr_interface = XRServer::get_singleton()->find_interface("OpenXR");
```

Comment:
Is the `OpenXR` interface available when in panel mode?

On Android, we pass `--xr-mode off` when launching the panel, so I'd expect OpenXR to be unavailable. Is that different on desktop?

Reply:
> Is the `OpenXR` interface available when in panel mode?

Yes. If the engine was compiled with OpenXR support, then the interface will _exist_, so `find_interface("OpenXR")` will return it. But if `--xr-mode off` was passed, then it won't be initialized.

I'm using this as a way to determine if OpenXR is even a possibility on the given platform. If you were running on iOS or the Web, then `find_interface("OpenXR")` would return a null reference.